CLA_SIGNERS.yaml

```
@@ -24,6 +24,9 @@ people:
   - name: Bramford Horton
     email: bram.horton@gmail.com
     github: bramford
+  - name: Darwin Yip
+    email: yip.darwin@gmail.com
```

Possible to use this email instead?
```suggestion
    email: darwinyip@users.noreply.github.com
```